src/sentry/api/endpoints/group_tags.py

```diff
@@ -5,18 +5,25 @@
 from sentry import tagstore
 from sentry.api.base import EnvironmentMixin
 from sentry.api.bases.group import GroupEndpoint
+from sentry.api.serializers import serialize
 from sentry.models import Environment
 
 
 class GroupTagsEndpoint(GroupEndpoint, EnvironmentMixin):
     def get(self, request, group):
+
+        # optional queryparam `key` can be used to get results
+        # only for specific keys.
+        keys = [tagstore.prefix_reserved_key(k)
+                for k in request.GET.getlist('key') if k] or None
+
         try:
             environment_id = self._get_environment_id_from_request(
                 request, group.project.organization_id)
         except Environment.DoesNotExist:
-            data = []
+            tag_keys = []
         else:
-            data = tagstore.get_group_tag_keys_and_top_values(
-                group.project_id, group.id, environment_id)
+            tag_keys = tagstore.get_group_tag_keys_and_top_values(
+                group.project_id, group.id, environment_id, limit=9, keys=keys)
```

Comment:
What's the `limit=9` about?

Reply:
and yeah, this one is just copied from the hardcoded one in the `GroupTagKeyDetailsEndpoint` which always returns 9 top values for the tag distribution bars. This `GroupTagsEndpoint` acts as basically a multi-tag version of that one, but yeah it would be nice if it was more obvious what all these hardcoded limits do